Utilities/StorageFactory/scripts/edmStorageTrace.py

```diff
@@ -294,6 +294,63 @@ def summary(logEntries):
     for h, q in quantities:
         print_summary(h, q)
 
+####################
+# Read ranges
+####################
+def printReadRanges(logEntries):
+    for entry in logEntries:
+        if isinstance(entry, Entries.Read):
```

Comment:
Wouldn't making Entries.Read and Entries.Readv be classes with methods 'printReadRanges' more canonical?